src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(255) DEFAULT NULL,
```

Comment:
as columns name and country are nullable, there is no need to add DEFAULT NULL here, as they will be NULL by default